content/slides/data-inspection.qmd

```diff
@@ -18,35 +18,383 @@ the same datum using e.g. `gdalwarp`, `ogr2ogr`.
+            MEMBER["World Geodetic System 1984 (G1150)"],
+            MEMBER["World Geodetic System 1984 (G1674)"],
+            MEMBER["World Geodetic System 1984 (G1762)"],
+            MEMBER["World Geodetic System 1984 (G2139)"],
+            ELLIPSOID["WGS 84",6378137,298.257223563,
+                LENGTHUNIT["metre",1]],
+            ENSEMBLEACCURACY[2.0]],
+        PRIMEM["Greenwich",0,
+            ANGLEUNIT["degree",0.0174532925199433]],
+        ID["EPSG",4326]],
+    CONVERSION["US NSIDC Sea Ice polar stereographic north",
+        METHOD["Polar Stereographic (variant B)",
+            ID["EPSG",9829]],
+        PARAMETER["Latitude of standard parallel",70,
+            ANGLEUNIT["degree",0.0174532925199433],
+            ID["EPSG",8832]],
+        PARAMETER["Longitude of origin",-45,
+            ANGLEUNIT["degree",0.0174532925199433],
+            ID["EPSG",8833]],
+        PARAMETER["False easting",0,
+            LENGTHUNIT["metre",1],
+            ID["EPSG",8806]],
+        PARAMETER["False northing",0,
+            LENGTHUNIT["metre",1],
+            ID["EPSG",8807]]],
+    CS[Cartesian,2],
+        AXIS["easting (X)",south,
+            MERIDIAN[45,
+                ANGLEUNIT["degree",0.0174532925199433]],
+            ORDER[1],
+            LENGTHUNIT["metre",1]],
+        AXIS["northing (Y)",south,
+            MERIDIAN[135,
+                ANGLEUNIT["degree",0.0174532925199433]],
+            ORDER[2],
+            LENGTHUNIT["metre",1]],
+    USAGE[
+        SCOPE["Polar research."],
+        AREA["Northern hemisphere - north of 60°N onshore and offshore, including Arctic."],
+        BBOX[60,-180,90,180]],
+    ID["EPSG",3413]]
+Data axis to CRS axis mapping: 1,2
+FID Column = fid
+Geometry Column = geom
+ID: Integer64 (0.0)
+New Greenlandic: String (0.0)
+Old Greenlandic: String (0.0)
+Danish: String (0.0)
+Alternative placename: String (0.0)
+Object designation: String (0.0)
+Source: String (0.0)
+Municipality: String (0.0)
+Notes: String (0.0)
+Date of registration: String (0.0)
+Modification date: String (0.0)
+Authorized: String (0.0)
+Authorized date: String (0.0)
+Project manager: String (0.0)
+KMS_id: String (0.0)
+Latitude: Real (0.0)
+Longitude: Real (0.0)
+Card: String (0.0)
+Namer: String (0.0)
+Namesake: String (0.0)
+Meaning of the placename: String (0.0)
+Placename history: String (0.0)
+Danish explanation of Object designation: String (0.0)
+Greenlandic explanation of Object designation: String (0.0)
+English explanation of Object designation: String (0.0)
+Translation notes: String (0.0)
+label: String (0.0)
+```
+
+::: {.notes}
+* `-al`: All layers
+* `-so`: Summary only; don't list individual features.
+* GeoPackages (`GPKG`) are a special extension to SQLite databases.
+* There may be many layers, but this particular GeoPackage just has one called
+  "SELECT", which is a poor name.
+* Like `gdalinfo`, the CRS is in WKT. First thing to look for is a "top-level"
+  EPSG code.
```

Comment:
This is handy, and will usually be available, but not all CRSs are going to necessarily have an EPSG code. Still, not a bad idea to look for this when evaluating the metadata for a CRS.

Reply:
Yeah, I wasn't sure what to suggest as a backup course of action in case a well-known ID isn't available. What do you think we should prescribe?

Reply:
We think we'll cover this in the previous slide deck by adding a slide like "representing geospatial metadata" discussing EPSG codes, WKT, Proj, ...?